content/cloud-docs/api-docs/team-access.mdx

```diff
@@ -214,6 +216,7 @@ Properties without a default value are required.
 | `data.attributes.state-versions`         | string  | "none"  | If `access` is `custom`, the permission to grant for the workspace's state versions. Can only be used when `access` is `custom`. Valid values include `none`, `read-outputs`, `read`, or `write`. |
 | `data.attributes.sentinel-mocks`         | string  | "none"  | If `access` is `custom`, the permission to grant for the workspace's Sentinel mocks. Can only be used when `access` is `custom`. Valid values include `none`, or `read`.                          |
 | `data.attributes.workspace-locking`      | boolean | false   | If `access` is `custom`, the permission granting the ability to manually lock or unlock the workspace. Can only be used when `access` is `custom`.                                                |
+| `data.attributes.run-tasks`      | boolean | false   | If `access` is `custom`, the permission granting the ability to manage workspace run tasks. Can only be used when `access` is `custom`.                                                |
```

Comment:
Hrmm yeah.  It was mostly a copy-paste of above..

Perhaps:

```
Can only be used when `access` is `custom`. This permission grants the Team to manage workspace run tasks.  
```

"workspace run tasks"  would be a link to https://www.terraform.io/cloud-docs/workspaces/settings/run-tasks

Reply:
ahhh okay how about "This permission allows the team to manage run tasks within the workspace". OR is this "Run tasks within the organization" --> like is this "you can create, delete, etc run tasks for everyone, or just within a specific workspace?

Reply:
> like is this "you can create, delete, etc run tasks for everyone, or just within a specific workspace?
Team Access only affects access within that workspace.

I definitely think we need the "If `access` is `custom`, " bit because for the API it's an important prerequisite that the person needs to know.  so perhaps:

```
If `access` is `custom`, this permission allows the team to manage run tasks within the workspace
```

or if we want to put the custom bit at the end.

```
This permission grants the Team to manage workspace run tasks. Can only be used when `access` is `custom`.
```

I'm personally not a fan of that style.  Because it _feels_ like I'm breaking the flow of sentence i.e.

I prefer "Under this circumstance you can use this thing" instead of "You can use this thing but only under this circumstance". 

But I could be very wrong!

Reply:
Fixed